cdap-common/src/main/java/co/cask/cdap/common/HttpExceptionHandler.java

```diff
@@ -64,13 +64,13 @@ public void handle(Throwable t, HttpRequest request, HttpResponder responder) {
       responder.sendString(HttpResponseStatus.NOT_IMPLEMENTED, t.getMessage());
     } else {
       LOG.error("Unexpected error: request={} {} user={}:", request.getMethod().getName(), request.getUri(),
-                SecurityRequestContext.getUserId().or("<null>"), t);
+                SecurityRequestContext.getUserId(), t);
```

Comment:
I  see what you mean. Before when it userid was null as a string then the message will be user={null} and when it was actually null the message would be user={<null>} I will fix it for the same.